package.json

```diff
@@ -117,6 +117,7 @@
     "sqlite3": "^5.0.1",
     "tap-spec": "^5.0.0",
     "tape": "^5.1.1",
+    "tedious": "9.2.3",
```

Comment:
Can we bump it to 11? Which version do you use for running in prod currently?

Reply:
This is the version that we have been running in prod.  Our application stack has thousands of connections to Azure SQL, and Tedious 9 allows for token auth.  It makes sense to be in line with the latest releases